browser/lib/findNoteTitle.js

```diff
@@ -0,0 +1,32 @@
+export function find (value) {
```

Comment:
[Question] Why didn't you name this method like `findNoteTitle`?
I think `find` is a little ambiguous.